### PR TITLE
Fix example in src/object.js

### PR DESCRIPTION
**Pull request by DamienCassou**

### CODE REVIEW

File: src/object.js [hunk 1]
```diff
@@ -39,7 +40,8 @@ define([
 	 * Creating instances:
 	 *
 	 *    var milou = dog({name: milou});
-	 *    milou.say('hello Tintin'); ;; => 'Woof Woof, hello Tintin'
+	 *    console.log(milou.say('hello Tintin')); ;; => 'Woof Woof, hello Tintin'
```
Comment (BenjaminVanRyseghem) [inline src/object.js]: I know @NicolasPetton *loves* tu use `;;` for comments, but it's still `//` in JavaScript :smile:

Comment (BenjaminVanRyseghem) [inline src/object.js]: @DamienCassou it's quite common in JS examples to omit the `console.log` for readability.
The value in the comment is the value returned, not printed

**Comment by DamienCassou** [comment]
Thanks for the feedback. Fixed.

**Comment by BenjaminVanRyseghem** [comment]
Thanks for you contribution :+1:

**Pull request merged**